Ghidra/Processors/8048/data/languages/8048.slaspec

```diff
@@ -434,7 +434,7 @@ RniI: Imm is Imm {
   goto Addr12;
 }
 :JMPP AddrInd   is ophi=11 & oplo=3 & AddrInd {
-  goto AddrInd;
+  goto [AddrInd];
```

Comment:
What do the brackets do here?  It looks like you are adding a new indirection, but that can't be the case because as you say there was already one too many?

Reply:
I'm not quite qualified to give a proper answer, but it came from https://github.com/NationalSecurityAgency/ghidra/discussions/4818 .

I can only quote the sleigh docs `7.7.2.5. Branching Statements` :

> The operators BRANCHIND, CALLIND, and RETURN all have the same semantic meaning and all use the same syntax to specify an indirect address. 
> [...]
> ```
> :ret is opcode=8 {
>     sp=sp+4;
>     tmp:4 = * sp;
>     return [tmp];
> }
> ```
> 
> Square brackets surround the varnode containing the address.

Reply:
I'm mostly curious what is going on since for example in the case of all the others jumps/branches, I have _no_
brackets in the goto, but I _do_ have `*[CODE]:1` in the exports of `Addr8` and `Addr12`, which seems to work fine?  That is also how the 8051 slaspec does it.

According to the sleigh docs, it seems the syntax with `goto []` requires the space of the destination instruction to be the same as the space of the branch instruction, which is of course the case here, but it would be interesting to know if `JMPP` can be made to work with a `*[]:1` indirection like regular `JMP`, so that the space can be explicitly specified...